content/applications/sales/sales/invoicing/expense.rst

```diff
@@ -1,65 +1,228 @@
-================================
-Re-invoice expenses to customers
-================================
+===============================
+Reinvoice expenses to customers
+===============================
 
-It often happens that your employees have to spend their personal money
-while working on a project for your client. Let's take the example of a
-consultant paying for a hotel to work on the site of your client. As a
-company, you would like to be able to invoice that expense to your
-client.
+While working on a project for a client, employees often have to spend their own money on various
+expenses related to the project.
 
-Expenses configuration
-======================
+For example, an employee may need to use their own money to pay for a hotel, while they provide an
+on-site service for a client. As a company, this expense should be reinvoiced to the customer. With
+Odoo, these kinds of expenses can quickly be reinvoiced to the customer related to the project.
 
-To track & invoice expenses, you will need the expenses app. Go to
-:menuselection:`Apps --> Expenses` to install it.
+Expenses application
+====================
 
-You should also activate the analytic accounts feature to link expenses
-to the sales order, to do so, go to :menuselection:`Invoicing -->
-Configuration --> Settings` and activate *Analytic Accounting*.
+To be able to reinvoice a customer for an expense, the *Expenses* application **must** be installed.
 
-Add expenses to your sales order
-================================
+To install the *Expenses* application, navigate to :menuselection:`Main Odoo Dashboard --> Apps`,
```

Comment:
No need to capitalize _main Odoo dashboard_.

And just a note, in other PRs you've written this as "Odoo main dashboard", I think both are OK, just wanted to point this out if there is a reason behind it 🙂
```suggestion
To install the *Expenses* application, navigate to :menuselection:`main Odoo dashboard --> Apps`,
```